apps/passport-server/test/devconnect.spec.ts

```diff
@@ -2945,13 +2959,19 @@ describe("devconnect functionality", function () {
       expect(
         await poapService.getPoapClaimUrlByTicketId("1", "zuzalu23")
       ).to.eq(TEST_POAP_LINK_1);
+      expect(
+        await poapService.getPoapClaimUrlByTicketId("1", "zuconnect")
+      ).to.eq(TEST_POAP_LINK_1);
```

Comment:
recalling from the last PR that this behavior is still a bit sketchy to me, but should be OK for now I guess.